_dashboards/discover/index-discover.md

```diff
@@ -0,0 +1,104 @@
+
+![Exploring data fields interface]({{site.url}}{{site.baseurl}}/images/discover-data-fields.png)
+
+Follow these steps to explore the data fields:
+
+1. View the list of **Available fields**.
+1. Choose **Cancelled** to view the values (`true` and `false`).
+1. Choose the plus (+) sign to add the field to the document table. You'll see this field automatically add to **Selected fields** and the document table.  
+1. Select **FlightDelay** from the **Available fields** list, and then choose the plus (+) sign to add the field to the document table.
+1. Optional: Rearrange the table columns by selecting the table header and then choosing **Move left** or **Move right**.
+
+## Searching data
+
+You can use the search toolbar or enter a DQL query in the **DevTools** console to search data in Dashboards, as shown in the following image. The search toolbar is best for basic queries, such as searching by a field name. DQL is best for complex queries, such as searching data using a term, string, boolean, date, range, or nested query.
+
+![Searching data interface]({{site.url}}{{site.baseurl}}/images/discover-search.png)
+
+Follow these steps to search data:
+
+1. In the search toolbar, enter the boolean query. For example, `FlightDelay:true AND FlightDelayMin >= 60` to search the data for flights delayed 60 minutes or longer.
+2. Choose **Update**.
+3. Optional: Choose the arrow (`>`) in a table row to expand the row and view the document table details.
+
+## Filtering data
+
+Filters allow you to refine sets of documents to subsets of those documents. For example, you can filter data to include or exclude certain fields, as shown in the following image.
+
+![Filtering data interface]({{site.url}}{{site.baseurl}}/images/discover-filter.png)
+
+Follow these steps to filter data:
+
+1. In the filter bar, choose **Add filter**.
+1. Select options from the **Field** > **Operator** > **Value** dropdown lists, for example, `Cancelled > is > true`.
+1. Choose **Save**.
+2. To remove the filter, choose the close icon (x) next to filter name.
+3. Optional: Add more filters to further explore the data.  
+
+## Analyzing data in the document table
+
+You can view the document table fields to better understand the data and gather insights for more informed decision-making. 
+
+1. Choose the arrow icon (>) to expand a table row.
+2. View the fields and details.
+3. Switch between **Table** and **JSON** tabs to view the different formats, for example, as shown in the following image.  
+
+![Analyzing data in the document table]({{site.url}}{{site.baseurl}}/images/discover-analyze.png)
+
+## Saving the search
+
+You can save your search to use it later, generate a report, or build visualizations and dashboards. Saving a search saves the query text, filters, and current data view.  
+
+1. Choose the save icon in the toolbar. 
```

Comment:
Introduce the list with a brief sentence ending in a colon.

Reply:
Rearranged sentences to address comment.